indico/modules/events/papers/controllers/display.py

```diff
@@ -53,3 +61,114 @@ def _process(self):
 class RHPaperTimeline(RHPapersBase):
     def _process(self):
         return NotImplementedError
+
+
+class RHDisplayJudgingArea(RHPapersBase):
+    def _checkProtection(self):
```

Comment:
Move that to a base class that checks whether the guy has judgment rights and inherit all the judgment-related RHs from that